extras/foreman_protector/dnf/foreman-protector.py

```diff
@@ -4,6 +4,8 @@
 
 import configparser
 
+PROTECT_COMMANDS = ('install', 'downgrade', 'reinstall', 'distro-sync', 'swap', 'upgrade', 'upgrade-minimal')
```

Comment:
tbh not sure about the commands here. given `f-m` only has wrappers for install and upgrade, the others *seem* wrong here, but then again they too can introduce issues…